### PR TITLE
🦺 #85: clinical profile zod schema

### PR DESCRIPTION
**Pull request by samrichca**
- fix failing tests due to IDs being too short
- add zod schemas for clinical profile
- add tests

### CODE REVIEW

File: packages/types/src/entities/ClinicalProfile.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { NanoId } from './NanoId.js';
+
+export const ClinicalProfile = z
+	.object({
+		ancestry: Ancestry,
+		birthSex: BirthSex,
+		clinicalProfilePrivateKey: NanoId,
+		familyHistoryOfCancer: HistoryOfCancer,
+		gender: Gender,
+		geneticsClinicVisited: GeneticsClinic.optional(),
```
Comment (alyssadsouza) [inline packages/types/src/entities/ClinicalProfile.ts:36]: isn't this required? I think the Prisma schema has it as required

**Review by alyssadsouza — COMMENTED** [review]
Looks good to me!

**Review by anncatton — APPROVED** [review]
Great, thanks for the new pr @samrichca 

If @alyssadsouza is good with this i think we can merge 🚀

File: packages/types/src/entities/ClinicalProfile.ts (hunk 1 above)
Comment (anncatton) [reply packages/types/src/entities/ClinicalProfile.ts:36]: yup good catch @alyssadsouza , should be required

Comment (samrichca) [reply packages/types/src/entities/ClinicalProfile.ts]: fixed

**Review by alyssadsouza — APPROVED** [review]
Looks good! 🚀 🚀 🚀

**Pull request merged**